terraform/environments/prd/aws/rds_varibles.tf

```diff
@@ -0,0 +1,30 @@
+locals {
+  gigachad_rds = {
+    engine              = "postgres"
+    storage             = "5"
+    db_subnet_name      = "subnet-db"
+    publicly_accessible = false
+    name                = "gigachad"
+    engine_version      = "14.3"
+    instance_class      = "db.t3.micro"
+    username            = "giga"
+    password            = "giga!sdf123."
```

Comment:
lets dynamically create password in the module. There is a random_password resource for this https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password